torch/onnx/_type_utils.py

```diff
@@ -170,7 +170,9 @@ def from_value(
             SymbolicValueError: when value.type()'s info are empty and default is None
         """
 
-        if not isinstance(value, (torch._C.Value, torch.Tensor)):
+        if not isinstance(value, (torch._C.Value, torch.Tensor)) or (
+            isinstance(value, torch._C.Value) and value.node().mustBeNone()
```

Comment:
Just to confirm, when a value must be none, value.type().scalarType() will be None?

Reply:
Why is it not handled by line 209?